<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dynamicOutput</class>
 <widget class="QWidget" name="dynamicOutput">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>130</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>488</width>
    <height>128</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>450</width>
     <height>121</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>450</width>
     <height>120</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>481</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="title">
    <string>GroupBox</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0">
       <widget class="QSpinBox" name="pwm_spinBox">
        <property name="minimumSize">
         <size>
          <width>90</width>
          <height>27</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSlider" name="pwm_slider">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="onButton">
          <property name="styleSheet">
           <string notr="true">QPushButton:checked, QPushButton:pressed{
	background-color: rgb(0, 85, 0);
}</string>
          </property>
          <property name="text">
           <string>Pin ON</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="offButton">
          <property name="styleSheet">
           <string notr="true">QPushButton:checked, QPushButton:pressed{
	background-color: rgb(170, 0, 0);
}</string>
          </property>
          <property name="text">
           <string>Pin OFF</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <widget class="QLCDNumber" name="lcdNumber">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>28</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 200, 0);</string>
        </property>
        <property name="intValue" stdset="0">
         <number>42</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>451</x>
     <y>0</y>
     <width>41</width>
     <height>118</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetMaximumSize</enum>
    </property>
    <item>
     <widget class="QCommandLinkButton" name="delButton">
      <property name="maximumSize">
       <size>
        <width>35</width>
        <height>30</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCommandLinkButton" name="upButton">
      <property name="maximumSize">
       <size>
        <width>35</width>
        <height>39</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCommandLinkButton" name="downButton">
      <property name="maximumSize">
       <size>
        <width>35</width>
        <height>39</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
